genai-perf/tests/test_rankings_converter.py

```diff
@@ -24,29 +24,45 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-from genai_perf.inputs.converters.rankings_converter import RankingsConverter
+import pytest
+from genai_perf.inputs.converters import RankingsConverter
 from genai_perf.inputs.input_constants import ModelSelectionStrategy, OutputFormat
 from genai_perf.inputs.inputs_config import InputsConfig
+from genai_perf.inputs.retrievers.generic_dataset import GenericDataset, DataRow, FileData
+from typing import List, Optional
 
 
 class TestRankingsConverter:
 
+    @staticmethod
+    def create_generic_dataset(
```

Comment:
It looks like each of these classes implements its own `create_generic_dataset`, is the idea that each converter makes generic datasets a little differently for their own testing?

Reply:
The generic dataset should theoretically be the same, but they all are testing slightly different things (e.g. rankings wants a multi-file setup with query and passage files, some tests want single/multiple images, some tests want single/multiple texts, etc.). Your comments is leading me to think it's likely there's a refactor opportunity to create a mock class for GenericDataset that gets reused.

For now, I just have this function as a fixture that sets up the generic datasets in each test so that we can test just the converters in isolation without making any calls to the retrievers or inputs.